app/assets/stylesheets/components/_card.scss

```diff
@@ -0,0 +1,44 @@
+.card {
```

Comment:
dont create class name that clashes with the bootstrap class

use like .nft-card instead